packages/uniforms-antd/package.json

```diff
@@ -32,8 +32,10 @@
     "src/*.tsx"
   ],
   "peerDependencies": {
-    "@ant-design/icons": "^4.0.0",
-    "antd": "^4.0.0",
+    "@ant-design/icons": "^5.0.0",
+    "antd": "^5.0.0",
+    "dayjs": "^1.11.13",
+    "rc-input-number": "^9.2.0",
```

Comment:
Fixes
```
The inferred type of 'default' cannot be named without a reference to '.pnpm/rc-input-number@9.2.0_react-dom@18.3.1_react@18.3.1__react@18.3.1/node_modules/rc-input-number'. This is likely not portable. A type annotation is necessary.ts(2742)
```

Reply:
Let's do this differently. 

1. Keep `rc-input-number@9.2.0` only in `devDependencies`
2. Add `/// <reference types="rc-input-number" />` to the `NumField` file